tools/cert_create/Makefile

```diff
@@ -10,6 +10,7 @@ V		:= 0
 DEBUG		:= 0
 BINARY		:= ${PROJECT}${BIN_EXT}
 OPENSSL_DIR	:= /usr
+USE_TBBR_DEFS	?= 1
```

Comment:
OK, I misunderstood what this code is doing. It can stay as-is but It would be good to add a comment explaining that the value passes from the top-level makefile is used by default.

Also, the user guide says "Specifying the platform is mandatory since the tool is platform specific." This is no longer true so could do with updating.